quesma/v2/core/routes/paths.go

```diff
@@ -7,25 +7,26 @@ import (
 )
 
 const (
-	GlobalSearchPath      = "/_search"
-	IndexSearchPath       = "/:index/_search"
-	IndexAsyncSearchPath  = "/:index/_async_search"
-	IndexCountPath        = "/:index/_count"
-	IndexDocPath          = "/:index/_doc"
-	IndexRefreshPath      = "/:index/_refresh"
-	IndexBulkPath         = "/:index/_bulk"
-	IndexMappingPath      = "/:index/_mapping"
-	FieldCapsPath         = "/:index/_field_caps"
-	TermsEnumPath         = "/:index/_terms_enum"
-	EQLSearch             = "/:index/_eql/search"
-	ResolveIndexPath      = "/_resolve/index/:index"
-	ClusterHealthPath     = "/_cluster/health"
-	BulkPath              = "/_bulk"
-	AsyncSearchIdPrefix   = "/_async_search/"
-	AsyncSearchIdPath     = "/_async_search/:id"
-	AsyncSearchStatusPath = "/_async_search/status/:id"
-	KibanaInternalPrefix  = "/.kibana_"
-	IndexPath             = "/:index"
+	GlobalSearchPath          = "/_search"
+	IndexSearchPath           = "/:index/_search"
+	IndexAsyncSearchPath      = "/:index/_async_search"
+	IndexCountPath            = "/:index/_count"
+	IndexDocPath              = "/:index/_doc"
+	IndexRefreshPath          = "/:index/_refresh"
+	IndexBulkPath             = "/:index/_bulk"
+	IndexMappingPath          = "/:index/_mapping"
+	FieldCapsPath             = "/:index/_field_caps"
+	TermsEnumPath             = "/:index/_terms_enum"
+	EQLSearch                 = "/:index/_eql/search"
+	ResolveIndexPath          = "/_resolve/index/:index"
+	ClusterHealthPath         = "/_cluster/health"
+	BulkPath                  = "/_bulk"
+	AsyncSearchIdPrefix       = "/_async_search/"
+	AsyncSearchIdPath         = "/_async_search/:id"
+	AsyncSearchStatusPath     = "/_async_search/status/:id"
+	KibanaInternalPrefix      = "/.kibana_"
+	IndexPath                 = "/:index"
+	ExecutePainlessScriptPath = "/_scripts/painless/_execute"
```

Comment:
How full workflow looks like? Is this first request from client to send some context information?

Reply:
When editing a script kibana provides a playground.   Kibana sends the script and sample data and renders the result. 

<img width="1020" alt="Screenshot 2024-12-06 at 11 59 31" src="https://github.com/user-attachments/assets/9c6bb42c-4c7c-4146-b3ed-8804d5cba8df">